src/common/image.c

```diff
@@ -272,7 +272,7 @@ void dt_image_path_append_version(int imgid, char *pathname, size_t pathname_len
 
 void dt_image_print_exif(const dt_image_t *img, char *line, size_t line_len)
 {
-  if(img->exif_exposure >= 0.1f)
+  if(img->exif_exposure >= 1.0f)
     snprintf(line, line_len, "%.1f'' f/%.1f %dmm iso %d", img->exif_exposure, img->exif_aperture,
```

Comment:
The notation for seconds is '' (two), just one means minutes:

http://english.stackexchange.com/questions/114205/english-notation-for-hour-minutes-and-seconds

I didn't touch that but I guess that was the rationale behind it.